### PR TITLE
Compiles successfully on armv7 (raspberry pi).

### PR DESCRIPTION
**Pull request by yakman2020**
@Microsoft/omi-devs a Runs regress successfully. Need to pbuild before merge but don't expect any problems.

### CODE REVIEW

**Comment by JumpingYang001** [comment]
OMI will support mobile phone, people will use OMI as a client or server in their phone?

**Comment by yakman2020** [comment]
In order to support mobile we would have to have an ARM package for android. This just makes it work on ARM hosts running linux.  That is interesting because there are a bunch of IoT hosts running linux on the arm.

Anyway, just one more architecture.

**Comment by yakman2020** [comment]
@Microsoft/omi-devs  Passed pbuild

File: Unix/buildtool [hunk 1]
```diff
@@ -966,7 +976,7 @@ if [ "$arg1" = "cshlibflags" -o "$arg1" = "cxxshlibflags" ]; then
     r=""
 
     case "$platform" in
-        LINUX_IX86_GNU|LINUX_X86_64_GNU|LINUX_PPC_GNU)
+        LINUX_IX86_GNU|LINUX_X86_64_GNU|LINUX_PPC_GNU|LINUX_ARM_GNU)
```
Comment (jeffaco) [inline Unix/buildtool]: These are duplicated so many times, I wonder if you'd be better off defining a GENERIC_LINUX symbol earlier and then just testing that. Then, in the future, if we need to add a platform, we add it in one place?

**Pull request merged**